patches/chrome-common-BUILD.gn.patch

```diff
@@ -10,19 +10,19 @@ index 195f082ff6d37c19e37d18170909126c0a8fc47b..845994e88564e1ed8ad5ceda57f17e3a
    public_deps = [
      "//base",
      "//components/version_info",
-@@ -152,6 +153,7 @@ static_library("common") {
-   ]
- 
-   public_deps = [
-+    "//brave/common",
-     ":available_offline_content_mojom",
-     ":buildflags",
-     ":channel_info",
-@@ -545,6 +547,7 @@ static_library("non_code_constants") {
+@@ -545,6 +546,7 @@ static_library("non_code_constants") {
      "//printing/buildflags",
      "//ui/base:buildflags",
    ]
 +  configs += [ "//brave/common:constants_configs" ]
  }
  
  component("chrome_features") {
+@@ -634,6 +636,7 @@ static_library("constants") {
+     deps += [ "//chromeos/crosapi/cpp" ]
+   } else if (is_linux || is_chromeos) {
+     sources += [ "chrome_paths_linux.cc" ]
++    deps += [ "//brave/common:channel_info" ]
```

Comment:
existing patch `deps += [ "//brave/common:channel_info" ]` should be a public dep and then this is not needed

Reply:
@bridiver Good point. Didn't see the ":channel_info" in deps just a few lines above the existing patch, but agreed making that a public_dep in the "//chrome/common:channel_info" target is what we want.

Updated, PTAL again, thanks!